package.json

```diff
@@ -112,6 +112,7 @@
     "react-dom": "^16.8.6",
     "react-element-to-jsx-string": "^14.0.2",
     "react-hot-loader": "^4.8.2",
+    "react-markdown": "^4.0.8",
```

Comment:
@layershifter @kuzhelov @miroslavstastny 
- are we ok with adding this dependency?
- can we suggest some existing tooling to achieve this?

Reply:
@layershifter added this

Reply:
> can we suggest some existing tooling to achieve this?

Nope

> are we ok with adding this dependency?

Yep, it's only for docs